declarations/MCVersionsManager.d.ts

```diff
@@ -0,0 +1,28 @@
+import { VersionDownloadDetails } from './Common';
+
+export interface MCVMInterface {
+    getVersionManifest(runtime: string): Promise<VersionManifest>;
+}
+
+export interface VersionsManifest {
+    versions: Array<VersionDetails>;
+}
+
+export interface VersionDetails {
+    id: string;
+    type: string;
+    url: string;
+    time: string;
+    releaseTime: string;
+}
+
+export interface VersionsDict {
+    [key: string]: VersionDetails;
+}
```

Comment:
This works but probably more intuitive as a Record.

```export type VersionsDict = Record<string, VersionDetails>```

See: https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt

Reply:
Oh that's super cool, will update, tyty